### ISSUE TITLE
NFS DB error even after deleting all db files

### ISSUE THREAD

**Issue by bandrel** [Pennyw0rth/NetExec#432]
When running `nxc nfs` for the first time the following error is displayed. The file is deleted and when executing the file is recreated but the error persists. 
```
# nxc nfs <target>

                    [-] Error reflecting tables for the NFS protocol - this means there is a DB schema mismatch
                    [-] This is probably because a newer version of nxc is being run on an old DB schema
                    [-] Optionally save the old DB data (`cp /root/.nxc/workspaces/default/nfs.db ~/nxc_nfs.bak`)
                    [-] Then remove the NFS DB (`rm -f /root/.nxc/workspaces/default/nfs.db`) and run nxc to initialize the new DB
```
**To Reproduce**
Steps to reproduce the behavior i.e.:
Command: `nxc nfs <target>`

Resulted in:
```

                    [-] Error reflecting tables for the NFS protocol - this means there is a DB schema mismatch
                    [-] This is probably because a newer version of nxc is being run on an old DB schema
                    [-] Optionally save the old DB data (`cp /root/.nxc/workspaces/default/nfs.db ~/nxc_nfs.bak`)
                    [-] Then remove the NFS DB (`rm -f /root/.nxc/workspaces/default/nfs.db`) and run nxc to initialize the new DB
```

**NetExec info**
 - OS: Ubuntu 22.04.5 
 - Version of nxc: 1.2.0 - ItsAlwaysDNS - 27313a0
 - Installed from: pipx install git+https://github.com/Pennyw0rth/NetExec --force

**Comment by NeffIsBack** [comment]
Yep that's a bug! Thanks for reporting, let me check what is wrong here! In the meantime you can solve it by removing the entire `~/.nxc` folder if you have no data to lose.
![image](https://github.com/user-attachments/assets/edd3f247-ae41-4cf5-b5e6-bde9140db061)

**Comment by termanix** [comment]
You can run this command until fixing the bug. It will fix it.

`rm -rf /root/.nxc/workspaces`

**Comment by NeffIsBack** [comment]
Should be fixed with #433
